docs/src/pages/customization/unstyled-components/unstyled-components.md

```diff
@@ -0,0 +1,139 @@
+# Unstyled components
+
+<p class="description">Implementing a custom design system using Material-UI</p>
+
+Material-UI unstyled provides a set of components without any styles. These can be used to implement a custom design system that is not based on Material Design.
+
+So far just a few components have been created,
+but we intend to focus on the unstyled package extensively in the upcoming months.
+
+## Use cases
+
+Is the unstyled package good for you?
+
+If you:
+
+- need to implement a design system that significantly differs from Material Design,
+- need precise control over the rendered HTML of the components,
+- or need to implement custom components but don't want to deal with accessibility features on your own,
+
+then you will find `material-ui/unstyled` useful.
+
+If, however, you:
+
+- need components that look good out of the box,
+- need to customize Material Design with your brand colors,
+- or need to implement a design system based on Material Design,
+
+then you may be better off using the `@material-ui/core` package and [customizing it] (/how-to-customize).
+
+## Components vs hooks
```

Comment:
If https://abbreviations.yourdictionary.com/is-vs-or-vs-the-correct-abbreviation-format.html is true:

```suggestion
## Components vs. hooks
```